### PR TITLE
Fix bullet list for Hugo

### LINKED ISSUE #8591: Issue with k8s.io/docs/reference/setup-tools/kubeadm/implementation-details/
**This is a...** 

- [x] Bug Report
- [ ] Feature Request

**Problem:**

The paragraph "Core design principles" looks like it's supposed to be an itemised list, however it is all rendered as regular text.
This maybe/probably has to do with the switch to hugo?

**Proposed Solution:**

Fix the source markdown code of that page.

**Page to Update:**
https://kubernetes.io/docs/reference/setup-tools/kubeadm/implementation-details/

### PR DESCRIPTION
**Pull request by mistyhacks**
Fixes #8591 

I also took away a level of depth in the list.

### CODE REVIEW

File: content/en/docs/reference/setup-tools/kubeadm/implementation-details.md [hunk 1]
```diff
@@ -78,7 +77,7 @@ for creating custom clusters.
 Kubeadm executes a set of preflight checks before starting the init, with the aim to verify preconditions and avoid common cluster startup problems.
 In any case the user can skip specific preflight checks (or eventually all preflight checks) with the `--ignore-preflight-errors` option.
 
-- [warning] If the Kubernetes version to use (specified with the `--kubernetes-version` flag) is at least one minor version higher than the kubeadm CLI version
+- [warning] If the Kubernetes version to use (specified with the `--kubernetes-version` flag) is at least one minor version higher than the kubeadm CLI version.
```
Comment (ajaeger) [inline content/en/docs/reference/setup-tools/kubeadm/implementation-details.md]: What about using the shortcode here? See https://kubernetes.io/docs/home/contribute/style-guide/#shortcodes

I know this is outside of the scope of your change...

Comment (mistyhacks) [reply content/en/docs/reference/setup-tools/kubeadm/implementation-details.md]: This isn't a warning, it's a listing of a type of error message. I thought the same thing and almost changed it. I forgot to revert the period, that's it.

Comment (ajaeger) [reply content/en/docs/reference/setup-tools/kubeadm/implementation-details.md]: Oh - indeed...

Comment (bradtopol) [reply content/en/docs/reference/setup-tools/kubeadm/implementation-details.md]: @mistyhacks.  Agreed! Good catch

**Pull request merged**